src/app/api/menu/route.ts

```
@@ -1,27 +1,36 @@
+ * @route GET /api/menu
+ * @header x-api-key - API 인증 키 (env: API_KEY)
+ * @query start - 조회 시작일 (YYYY-MM-DD)
+ * @query end - 조회 종료일 (YYYY-MM-DD)
+ * @returns 날짜 범위 내 메뉴 목록
+ */
 export async function GET(req: NextRequest) {
   const apiKey = req.headers.get('x-api-key');
-  if (apiKey !== process.env.API_KEY) return json({message: 'Unauthorized'}, 401);
+  const API_KEY = process.env.API_KEY;
+  if (!API_KEY || apiKey !== API_KEY)
+    return json({ error: 'Unauthorized' }, 401);
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)